sklearn/model_selection/_search.py

```diff
@@ -674,7 +674,9 @@ def fit(self, X, y=None, groups=None, **fit_params):
 
         base_estimator = clone(self.estimator)
 
-        parallel = Parallel(n_jobs=self.n_jobs, verbose=self.verbose,
+        # We don't pass verbose to Parallel, since _fit_and_score is detailed
+        # enough
```

Comment:
I'd say remove the comment

This is potentially the kid of comment that makes people think "why would you do that in the first place anyway" because the context is lost